tests/baselines/reference/tscWatch/moduleResolution/watches-for-changes-to-package-json-main-fields.js

```diff
@@ -178,127 +166,14 @@ Input::
 }
 
 
-Timeout callback:: count: 1
-1: timerToInvalidateFailedLookupResolutions *new*
+Before running Timeout callback:: count: 0
 
-Before running Timeout callback:: count: 1
-1: timerToInvalidateFailedLookupResolutions
-
-Host is moving to new time
-After running Timeout callback:: count: 1
-
-Timeout callback:: count: 1
-2: timerToUpdateProgram *new*
-
-Before running Timeout callback:: count: 1
-2: timerToUpdateProgram
-
-Host is moving to new time
 After running Timeout callback:: count: 0
-Output::
->> Screen clear
-[[90mHH:MM:SS AM[0m] File change detected. Starting incremental compilation...
-
-======== Resolving module 'pkg2' from '/user/username/projects/myproject/packages/pkg1/index.ts'. ========
-Module resolution kind is not specified, using 'Node10'.
-Loading module 'pkg2' from 'node_modules' folder, target file types: TypeScript, Declaration.
-Searching all ancestor node_modules directories for preferred extensions: TypeScript, Declaration.
-Directory '/user/username/projects/myproject/packages/pkg1/node_modules' does not exist, skipping all lookups in it.
-Directory '/user/username/projects/myproject/packages/node_modules' does not exist, skipping all lookups in it.
-Found 'package.json' at '/user/username/projects/myproject/node_modules/pkg2/package.json'.
-File '/user/username/projects/myproject/node_modules/pkg2.ts' does not exist.
-File '/user/username/projects/myproject/node_modules/pkg2.tsx' does not exist.
-File '/user/username/projects/myproject/node_modules/pkg2.d.ts' does not exist.
-'package.json' does not have a 'typesVersions' field.
-'package.json' does not have a 'typings' field.
-'package.json' does not have a 'types' field.
-'package.json' has 'main' field 'build/other.js' that references '/user/username/projects/myproject/node_modules/pkg2/build/other.js'.
-File name '/user/username/projects/myproject/node_modules/pkg2/build/other.js' has a '.js' extension - stripping it.
-File '/user/username/projects/myproject/node_modules/pkg2/build/other.ts' does not exist.
-File '/user/username/projects/myproject/node_modules/pkg2/build/other.tsx' does not exist.
-File '/user/username/projects/myproject/node_modules/pkg2/build/other.d.ts' exists - use it as a name resolution result.
-'package.json' does not have a 'peerDependencies' field.
-Resolving real path for '/user/username/projects/myproject/node_modules/pkg2/build/other.d.ts', result '/user/username/projects/myproject/packages/pkg2/build/other.d.ts'.
-======== Module name 'pkg2' was successfully resolved to '/user/username/projects/myproject/packages/pkg2/build/other.d.ts' with Package ID 'pkg2/build/other.d.ts@1.0.0'. ========
-[96mpackages/pkg1/index.ts[0m:[93m1[0m:[93m15[0m - [91merror[0m[90m TS2305: [0mModule '"pkg2"' has no exported member 'TheNum'.
```

Comment:
Package.json was watched in two scenarios before this change - 
1.  if package json was used to determine file's metadata - in node16 etc 
2.  if this affected already resolved module resolution

Because we arent doing 2 watch any more and node10 isnt using package.json - this discrepency is shown in the test which uses node10